src/renderer.ts

```diff
@@ -329,17 +323,17 @@ const richLinkStyles = css`
     ${darkModeCss`
         background-color: ${neutral[20]};
         color: ${neutral[60]};
-        a {
+
+        h1 {
             color: ${neutral[60]};
-            border-bottom: 0.0625rem solid ${neutral[60]};
+            border-bottom: none;
         }
     `}
 `;
 
-const RichLink = (props: { url: string; linkText: string; format: Format }): ReactElement =>
-    styledH('aside', { css: richLinkStyles },
-        h('h1', null, props.linkText),
-        h(Anchor, { href: props.url, format: props.format }, 'Read more'),
+const RichLink = (props: { url: string; linkText: string }): ReactElement =>
+    styledH('a', { href: props.url, css: richLinkStyles },
+        styledH('aside', null, [h('h1', null, props.linkText), 'Read more'])
```

Comment:
Would it be possible to swap the `a` and `aside` here? So that the top level element is an `aside` and the `a` lives within that?